packages/syft/src/syft/service/code/user_code.py

```diff
@@ -1933,3 +1933,15 @@ def migrate_usercode_v5_to_v4() -> list[Callable]:
         drop("origin_node_side_type"),
         drop("l0_deny_reason"),
     ]
+
+
+@migrate(UserCodeV5, UserCode)
```

Comment:
NIT: could we delete UserCodeV4 from the code?

Reply:
and also its corressponding migrations.

Reply:
as since we need to maintain migrations for only last two versions.

Reply:
ya I can, but it kind of seems like bad practice in a way to lose the ability to migrate from old versions. Especially since all of these updates to UserCode are within such a short timeframe (V4 was just two weeks ago), migrations would be broken from UserCode from less than a month ago with this approach. Going to raise in slack since I think it would be good to understand why we do it this way.